src/components/pages/home/elements/PhotoBlock.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import logo from '../../../../assets/images/logo.svg';
+
+export default class PhotoBlock extends Component {
+  render() {
+    return (
+      <div className="photo_block">
```

Comment:
CSS class names should be hyphenated rather than underscored, i.e. `photo-block`